admin/database/database.go

```diff
@@ -260,32 +267,36 @@ type UpdateOrganizationOptions struct {
 	QuotaSlotsTotal         int
 	QuotaSlotsPerDeployment int
 	QuotaOutstandingInvites int
+	QuotaNumUsers           int
+	QuotaManagedDataBytes   int64
+	BillingCustomerID       *string
 }
 
 // Project represents one Git connection.
 // Projects belong to an organization.
 type Project struct {
-	ID                   string
-	OrganizationID       string `db:"org_id"`
-	Name                 string
-	Description          string
-	Public               bool
-	CreatedByUserID      *string `db:"created_by_user_id"`
-	Provisioner          string
-	GithubURL            *string           `db:"github_url"`
-	GithubInstallationID *int64            `db:"github_installation_id"`
-	Subpath              string            `db:"subpath"`
-	ProdVersion          string            `db:"prod_version"`
-	ProdBranch           string            `db:"prod_branch"`
-	ProdVariables        map[string]string `db:"prod_variables"`
-	ProdOLAPDriver       string            `db:"prod_olap_driver"`
-	ProdOLAPDSN          string            `db:"prod_olap_dsn"`
-	ProdSlots            int               `db:"prod_slots"`
-	ProdTTLSeconds       *int64            `db:"prod_ttl_seconds"`
-	ProdDeploymentID     *string           `db:"prod_deployment_id"`
-	Annotations          map[string]string `db:"annotations"`
-	CreatedOn            time.Time         `db:"created_on"`
-	UpdatedOn            time.Time         `db:"updated_on"`
+	ID                     string
+	OrganizationID         string `db:"org_id"`
+	Name                   string
+	Description            string
+	Public                 bool
+	CreatedByUserID        *string `db:"created_by_user_id"`
+	Provisioner            string
+	GithubURL              *string           `db:"github_url"`
+	GithubInstallationID   *int64            `db:"github_installation_id"`
+	Subpath                string            `db:"subpath"`
+	ProdVersion            string            `db:"prod_version"`
+	ProdBranch             string            `db:"prod_branch"`
+	ProdVariables          map[string]string `db:"prod_variables"`
+	ProdOLAPDriver         string            `db:"prod_olap_driver"`
+	ProdOLAPDSN            string            `db:"prod_olap_dsn"`
+	ProdSlots              int               `db:"prod_slots"`
+	ProdTTLSeconds         *int64            `db:"prod_ttl_seconds"`
+	ProdDeploymentID       *string           `db:"prod_deployment_id"`
+	Annotations            map[string]string `db:"annotations"`
+	CreatedOn              time.Time         `db:"created_on"`
+	UpdatedOn              time.Time         `db:"updated_on"`
+	NextUsageReportingTime time.Time         `db:"next_usage_reporting_time"`
```

Comment:
`UsageReportedOn` seems like the time on which last usage was reported but In the code this is used as the reporting start time for next reporting window. This window may be bigger than one delta as reporting may be delayed because of data unavailability or other number of reasons. Do you still think this makes sense, so I will make the change